### PR TITLE
Translation for several board and some minor fix

### PR DESCRIPTION
**Pull request by wychlw**
- Fix and tweaked translation for LicheePi4A, LicheeCluster4A and LicheeConsole4A.

- Fix some link and format issue

### CODE REVIEW

**Comment by wychlw** [comment]
Mayby ready? I tried to correct where I can find out.

**Comment by KevinMX** [comment]
> Mayby ready? I tried to correct where I can find out.

Busy on other stuff these days. Will review later.

File: BeagleV-Ahead/Ubuntu/README.md [hunk 1]
```diff
@@ -1,65 +1,63 @@
...
-## 实际结果
+## Actual Results
 
-系统正常启动，成功通过板载串口登录。
+The system boots up successfully, and login via the onboard serial port is successful.
 
-### 启动信息
-
-屏幕录像（从刷写镜像到登录系统）：
+### Boot Messages
```
Comment (KevinMX) [inline BeagleV-Ahead/Ubuntu/README.md]: Inconsistent translation (Boot Messages / Boot Information / Boot Log?)

File: 100ASK/Melis/README.md [hunk 2]
```diff
@@ -46,36 +46,36 @@ KERNEL=="aw_efex[0-9]*", MODE="0666"
...
+System should boot up normally, with output visible through the serial port.
 
-## 实际结果
+## Actual Results
 
-系统正常启动，成功通过串口查看输出。
+System booted up successfully, and output was visible through the serial port.
 
-### 启动信息
+### Boot Information
```
Comment (KevinMX) [inline 100ASK/Melis/README.md]: I suggest "Boot log".

File: BeagleV-Fire/Ubuntu/README_zh.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+## 安装步骤
+
+### 刷写 Gateway 模块
+
+**理论上 Gateway 已经刷写好，开箱即用。**
+
+### 升级 Gateway
+
+**升级 Gateway 可直接在板上自带 Linux 中进行**，见[官方文档](https://docs.beagleboard.org/latest/boards/beaglev/fire/demos-and-tutorials/gateware/upgrade-gateware.html)
```
Comment (KevinMX) [inline BeagleV-Fire/Ubuntu/README_zh.md]: Gateway / Gateware?

File: CH32V103/Zephyr/README.md
Comment (KevinMX) [inline CH32V103/Zephyr/README.md:1]: Deleted?

File: BeagleV-Fire/Ubuntu/README.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
-- USB-C 电源适配器 / DC 电源一个
-- USB-UART 调试器一个
+- USB-C Power Adapter / DC Power Supply
+- USB-UART Debugger
 
-## 安装步骤
+## Installation Steps
 
-### 刷写 Gateway 模块
+### Flashing Gateway Module
```
Comment (KevinMX) [inline BeagleV-Fire/Ubuntu/README.md]: Gateway or Gateware?

File: LicheeCluster4A/Fedora/README.md [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
+### Hardware Information
 
 - Lichee Cluster 4A 8G / 16G
-- DC 12V 电源
+- DC 12V Power Supply
 - USB-A to A
-    - 或 LPi4A 底板
-- microSD 卡一张
-- 网络和网线（注意连接到 BMC 而非交换机）
+    - or LPi4A Baseboard
```
Comment (KevinMX) [inline LicheeCluster4A/Fedora/README.md]: Baseboard -> Dock/Motherboard?

File: LicheePi4A/README.md [hunk 6]
```diff
@@ -1,40 +1,41 @@
...
+    - Download link: [Fedora Images](https://openkoji.iscas.ac.cn/pub/dl/riscv/T-Head/th1520_light/images/)
+    - Reference Installation Document: [Fedora Installation Guide](https://fedoraproject.org/wiki/Architectures/RISC-V/T-Head)
 - Armbian
-    - 下载链接：https://github.com/chainsx/armbian-riscv-build/tree/main
-    - 参考安装文档：https://github.com/chainsx/armbian-riscv-build/blob/main/doc/licheepi-4a-install-guide.md
+    - Download link: [Armbian Images](https://github.com/chainsx/armbian-riscv-build/tree/main)
+    - Reference Installation Document: [Armbian Installation Guide](https://github.com/chainsx/armbian-riscv-build/blob/main/doc/licheepi-4a-install-guide.md)
 
-### 硬件开发板信息
+### Hardware Development Board Information
```
Comment (KevinMX) [inline LicheePi4A/README.md]: -> Board Information?

**Review by KevinMX — CHANGES_REQUESTED** [review]

**Pull request merged**